code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -3577,7 +3577,17 @@
 				if(prob(10))
 					to_chat(H,"<span class='warning'>You really don't feel very good.</span>")
 				if(prob(5))
-					H.adjustToxLoss(0.1)
+					switch(pick(1,2,3,4,5))
```

Comment:
You have reached the first stage of coder awareness @yackemflam 

1.  I CAN code this.

You have to grow to the next stage.

2.  I SHOULD code this.